apps/communications/contacts/elements/settings.html

```diff
@@ -34,28 +34,6 @@ <h1 data-l10n-id="settings"></h1>
               </button>
             </li>
           </ul>
-          <gaia-subheader id="fb-header" skin="organic">
-            <span data-l10n-id="facebook">Facebook</span>
-          </gaia-subheader>
-          <ul id="settingsFb" role="presentation" data-type="list" data-state="logged-out">
```

Comment:
Removing by now in the Mock. The idea is to remove all mocks of HTML and use the real DOM, but we are going to tackle this in a separate bug.